docs/user/introduction.rst

```diff
@@ -0,0 +1,57 @@
+Introduction
+============
+
+What is GeoMesa?
+----------------
+
+GeoMesa is an open-source, distributed, spatio-temporal database and suite of tools built on top
+of the Apache Accumulo (http://accumulo.apache.org/) column family store. It lets users quickly store, index, and query both stored and streaming geospatial data at scale. 
+
```

Comment:
we probably want to update this section to highlight the non-accumulo aspects of geomesa. @anthonyccri didn't we have some other verbiage somewhere?